### ISSUE TITLE
Code coverage reports

### ISSUE THREAD

**Issue by andrew-g-za** [interledger/java-ilp-core#38]
Is it worth adding code coverage reports to the repos? 
I see the interledger-js repos make use of https://codecov.io/

**Comment by adrianhopebailie** [comment]
Yes! Now that we have some tests I think we should definitely start tracking this

**Comment by andrew-g-za** [comment]
closed, see PR #39
